src/lib/Microsoft.Health.Fhir.R4.Ingest.Templates/Extensions/DateTimeExtensions.cs

```diff
@@ -8,7 +8,7 @@
 using EnsureThat;
 using Hl7.Fhir.Model;
 
-namespace Microsoft.Health.Extensions.Fhir
+namespace Microsoft.Health.Fhir.R4.Ingest.Templates.Extensions

```

Comment:
The project where this file was hosted currently targets net6.0. Since these methods are only used by the R4 templates I moved this class into the new `Health.Fhir.R4.Ingest.Tempates` project.